### ISSUE TITLE
Cannot find module  .node_moduleswebrtc-scalable-broadcastserver.js

### ISSUE THREAD

**Issue by rubo77** [muaz-khan/WebRTC-Scalable-Broadcast#3]
I did install it with npm version 1.4.21:

```
cd /var/www/
npm install webrtc-scalable-broadcast
```

When I start

```
node .\node_modules\webrtc-scalable-broadcast\server.js
```

I get the error:

```
module.js:340
    throw err;
          ^
Error: Cannot find module '/var/www/.node_moduleswebrtc-scalable-broadcastserver.js'
    at Function.Module._resolveFilename (module.js:338:15)
    at Function.Module._load (module.js:280:25)
    at Function.Module.runMain (module.js:497:10)
    at startup (node.js:119:16)
    at node.js:906:3
```

Do I have to [install nodes v1.0.1](http://stackoverflow.com/questions/30466668/install-nodes-npm-v1-0-1)?

**Comment by SLaks** [comment]
Linux paths use forward slashes, not backslashes.

**Comment by rubo77** [comment]
fixed by #4
